src/backend/gl/src/info.rs

```diff
@@ -124,10 +124,16 @@ fn get_string(gl: &gl::Gl, name: gl::types::GLenum) -> &'static str {
     }
 }
 
-fn get_usize(gl: &gl::Gl, name: gl::types::GLenum) -> usize {
+fn get_usize(gl: &GlContainer, name: gl::types::GLenum) -> Result<usize, Error> {
     let mut value = 0 as gl::types::GLint;
     unsafe { gl.GetIntegerv(name, &mut value) };
-    value as usize
+
+    let err = Error::from_error_code(unsafe { gl.GetError() });
```

Comment:
do these calls fail at all?

Reply:
Yes, I think it was failing with `MAX_TEXTURE_BUFFER_SIZE`, but I honestly can't remember.